bears/c_languages/ClangComplexityBear.py

```diff
@@ -85,10 +85,20 @@ def run(self, filename, file, max_complexity: int=8):
                 affected_code = (SourceRange.from_clang_range(cursor.extent),)
                 yield Result(
                     self,
-                    "The cyclomatic complexity of function {function} is "
-                    "{complexity} which exceeded maximal recommended value "
+                    "This function {function} should be simplified. The "
```

Comment:
i don't think this sounds good. You can't quite say: ``this function sum`` for example